### DISCUSSION TITLE
Knitting Error

### DISCUSSION THREAD

**Discussion by ewhittemore987** [LiNk-NY/IntroR#47] [General]
Hello everyone:

Does anyone know what the following error means?

<img width="812" alt="image" src="https://github.com/CUNYSPHCode/IntroR/assets/171465841/3073ce6d-4f65-4abe-bda8-a53e5925a971">

I had trouble knitting the Word file using the document I downloaded from Blackboard. However, when I created a new .rmd in R Studio, copied and pasted everything I had done in the existing file I downloaded (no changes), then knitted, there was no error and the knit to Word worked perfectly. 

Does anyone know what was causing the first one to throw the error?

**Comment by LiNk-NY** [comment]
Make sure that your source file is in a write-able folder. It may be that when you copied and pasted, you did so in a file location that was write-able and that's why it worked. 
Try moving your source file (.Rmd) to a write-able location, e.g., `Documents` folder.
-Marcel

**Reply by camilayepes16** [reply]
This happened to me when I tried to knit again while having the original document it created open! Once I closed that doc, it made a new one. Not sure if that was coincidental.

**Reply by LiNk-NY** [reply]
Ah, yes, you can't write while reading a file :+1:
